components/feature/media/src/main/java/mozilla/components/feature/media/state/MediaStateMachine.kt

```diff
@@ -14,6 +19,8 @@ import mozilla.components.support.base.log.logger.Logger
 import mozilla.components.support.base.observer.Observable
 import mozilla.components.support.base.observer.ObserverRegistry
 
+private const val DELAY_STATE_UPDATE = 250L
```

Comment:
Should we include also include in the unit of the time delays?

Reply:
Alright. I'll name it `DELAY_STATE_UPDATE_MS`.